src/app/components/PortraitVideoCarousel/PortraitVideoItem/index.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { PortraitVideoItemProps } from '#app/models/types/portraitVideoCarousel';
+import Heading from '../../Heading';
+import styles from './index.styles';
+
+export default (item: PortraitVideoItemProps) => {
+  const { images, headlines, id, onClick } = item;
+  const image = images?.[0]?.url;
+  const alt = images?.[0]?.altText || '';
```

Comment:
Let's use array & object destructuring (may need to rename some items too)

```suggestion
  const [image] = images;
  const { url, altText } = image;
```

Reply:
Hello, I've tried this approach, but it makes it a little tricky to read and cover all the optional edge cases.

Reply:
OK that's fair. Thanks for explaining.